scripts/returns/data-access.js

```diff
@@ -0,0 +1,124 @@
+const returnsConnector = require('../../src/lib/connectors/returns');
+
+/**
+ * Gets all returns that have the copiedForTesting flag in the metadata.
+ *
+ * @returns {Promise} Promise that will resolve with any test returns
+ */
+const getAllTestReturns = () => {
+  return returnsConnector.returns.findAll({
+    'metadata->>copiedForTesting': 'true'
+  });
+};
+
+/**
+ * Gets all versions for the return id
+ *
+ * @returns {Promise} Promise that will resolve with any versions
+ */
+const loadReturnVersions = returnId =>
+  returnsConnector.versions.findAll({ return_id: returnId });
+
+/**
+ * Deletes all return lines that have the given version ids.
+ *
+ * @param {Array} asyncversionIds The list of version ids to delete matching lines for
+ * @returns {Promise} A promise that will resolve to all deleted line calls
+ */
+const deleteLines = async versionIds => {
+  // get all the lines for the version ids
```

Comment:
Although I don't think it's documented, I think you should be able to delete multiple lines with a single request:
```
returnsConnector.lines.delete({
    version_id: {
      '$in': versionIds
    }
  });
```
I'll add an issue in Github on the hapi-pg-rest-api that delete many is not documented.